tools/windows-node-installer/test/e2e/aws_test.go

```diff
@@ -24,7 +25,7 @@ var (
 	// Microsoft Windows Server 2019 Base with Containers.
 	imageID      = "ami-0b8d82dea356226d3"
 	instanceType = "m4.large"
-	sshKey       = "libra"
+	sshKey       = "openshift-dev"
```

Comment:
currently in the aws portal libra key is replaced with openshift-dev, so that's why I made this change. Will change this back to libra as CI still haven't updated.

Reply:
okay.